build/Build.md

```diff
@@ -0,0 +1,190 @@
+# 1. Building a container image
```

Comment:
@DilanUA can you please remove the `build/.DS_Store` file?

Reply:
Done.